snapwell/snapecl.py

```diff
@@ -14,11 +14,75 @@
 
 
 import logging
+from math import inf, isinf, isnan, nan, sqrt
 
-from .snap_utils import Inf, Nan, dist, enterSnapMode, findKeyword, roundAwayFromEven
 from .snapconfig import OwcDefinition
 
 
+def dist(p1, p2):
```

Comment:
`math` should already have a `dist`?

Reply:
Indeed they do, fixed so that it uses `math.dist` .